src/health.h

```diff
@@ -127,149 +130,153 @@ typedef struct rrddimvar {
 
-    time_t db_after;                // the first timestamp evaluated by the db lookup
-    time_t db_before;               // the last timestamp evaluated by the db lookup
+    time_t db_after;                ///< the first timestamp evaluated by the db lookup
+    time_t db_before;               ///< the last timestamp evaluated by the db lookup
 
-    time_t delay_up_to_timestamp;   // the timestamp up to which we should delay notifications
-    int delay_up_current;           // the current up notification delay duration
-    int delay_down_current;         // the current down notification delay duration
-    int delay_last;                 // the last delay we used
+    time_t delay_up_to_timestamp;   ///< the timestamp up to which we should delay notifications
+    int delay_up_current;           ///< the current up notification delay duration
+    int delay_down_current;         ///< the current down notification delay duration
+    int delay_last;                 ///< the last delay we used
 
     // ------------------------------------------------------------------------
     // variables this alarm exposes to the rest of the alarms
 
-    RRDVAR *local;
-    RRDVAR *family;
-    RRDVAR *hostid;
-    RRDVAR *hostname;
+    RRDVAR *local;    ///< exposed to the rest of the alarms
+    RRDVAR *family;   ///< exposed to the rest of the alarms
+    RRDVAR *hostid;   ///< exposed to the rest of the alarms
+    RRDVAR *hostname; ///< exposed to the rest of the alarms
 
     // ------------------------------------------------------------------------
     // the chart this alarm it is linked to
 
-    struct rrdset *rrdset;
+    struct rrdset *rrdset; ///< set this is linked to
 
-    // linking of this alarm on its chart
+    /// linking of this alarm on its chart
     struct rrdcalc *rrdset_next;
+    /// linking of this alarm on its chart
     struct rrdcalc *rrdset_prev;
 
-    struct rrdcalc *next;
+    struct rrdcalc *next; ///< next in the list
 } RRDCALC;
 
 #define RRDCALC_HAS_DB_LOOKUP(rc) ((rc)->after)
 
-// RRDCALCTEMPLATE
-// these are to be applied to charts found dynamically
-// based on their context.
+/// RRDCALCTEMPLATE
+/// these are to be applied to charts found dynamically
+/// based on their context.
 typedef struct rrdcalctemplate {
-    char *name;
-    uint32_t hash_name;
+    char *name;         ///< name of the template
+    uint32_t hash_name; ///< hash of `name`
 
-    char *exec;
-    char *recipient;
+    char *exec;      ///< ktsaou: your help needed
+    char *recipient; ///< ktsaou: your help needed
 
-    char *context;
-    uint32_t hash_context;
+    char *context;         ///< ktsaou: your help needed
+    uint32_t hash_context; ///< hash of `context`
 
-    char *family_match;
-    SIMPLE_PATTERN *family_pattern;
+    char *family_match;             ///< ktsaou: your help needed
+    SIMPLE_PATTERN *family_pattern; ///< ktsaou: your help needed
```

Comment:
this is used to match the charts the alarm template should by applied to. Example `veth*` to match all `veth` interfaces.